plugins/scaffolder/src/components/TemplateCard/TemplateCard.tsx

```diff
@@ -72,7 +86,54 @@ const getTemplateCardProps = (
   };
 };
 
-export const TemplateCard = ({ template }: TemplateCardProps) => {
+const DeprecationWarning = () => {
+  const [anchorEl, setAnchorEl] = React.useState<SVGSVGElement | null>(null);
+
+  const handleClick = (event: React.MouseEvent<SVGSVGElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const styles = useDeprecationStyles();
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+
+  const Title = (
+    <Typography style={{ padding: 10, maxWidth: 300 }}>
+      This template syntax is deprecated. Please follow the documentation{' '}
+      <Link href="https://backstage.io/docs/features/software-templates/migrating-from-v1alpha1-to-v1beta2">
+        here
+      </Link>{' '}
+      to migrate
```

Comment:
Possible suggestion: 

> Please follow the **documentation to migrate**

...where the "documentation to migrate" is linked? Helps align to the best practice of never linking to "click here" or "here" :)

Reply:
Nice recommendation - will fix that now!